telegram/_message.py

```diff
@@ -1419,13 +1419,25 @@ def effective_attachment(
         .. versionchanged:: NEXT.VERSION
             :attr:`paid_media` is now also considered to be an attachment.
 
+        .. deprecated:: NEXT.VERSION
+            :attr:`successful_payment` will be removed in future major versions.
```

Comment:
the "major" is a bit confusing to me. Will it still be considered in *minor* version? same below.

Reply:
I took that wording from https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram%2F_passport%2Fpassportfile.py#L112

My understanding is that related to the stability policy the "next" major version removes it

Reply:
well, (git and) I will blame harshil, then :D I understand what it's supposed to mean, just stumbled on it on first read … If you like to keep it this way, I'll just merge :)

Reply:
Generally, I prefer not to mention specific versions beforehand - just gives use more freedom in case we need it …

Reply:
Then Id stick with this

Reply:
yea I put "major" versions cause of the stability policy